test/Npgsql.Tests/PgPassFileTests.cs

```diff
@@ -46,7 +64,14 @@ public void CreateTestFile()
 testhost:*:*:*:testdefaultpass
 # helpful comment goes here
 *:*:*:*:defaultpass";
+
+            // create a long filename with unusual allowed characters
+            const string uniqueFileName = "!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE!#$%&'()+,.;=@[]^_`{}~FREE.tmp";
```

Comment:
Is there any specific reason to test unusual allowed characters, what scenario exactly are you imagining? After all Npgsql is simply opening the file and not doing any sort of path manipulation etc.